### ISSUE TITLE
Laser cutter Z-axis gcode to spindle control

### ISSUE THREAD

**Issue by zapmaker** [zapmaker/GrblHoming#55]
User request: Is it possible to add better lasercutter support? For example, instead of driving the Z-Axis, there could be an option to automatically change the command to turn the Spindle Off/On.

Solution? do you have something in mind like a downward motion in Z means spindle on and upward spindle off?

**Comment by derFrickler** [comment]
Why don't you do that in you CAM? 
For my Laser i just set the Z Travel to minimum (0,1mm or so) and added M8/M9 commands before z down and after z up to switch the coolant pin of grbl.
I use ESTLCam where this can be easily configured.

That way i can use the same CAM for milling and lasering.

**Comment by zapmaker** [comment]
Good point. I am guessing the user has a CAM program with limited capabilities and wants Grbl Controller to do it. I think it is reasonable, but not highest priority.

**Comment by derFrickler** [comment]
Maybe an option to sent a custom GCode before Z Down and after Z Up would be an Optoin for grbl.
